src/test/java/org/zendesk/client/v2/model/TriggerTest.java

```diff
@@ -0,0 +1,64 @@
+package org.zendesk.client.v2.model;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.Test;
+
+import java.util.Arrays;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotNull;
+
+public class TriggerTest {
+
+    private Trigger parseJson(byte[] json) {
+        ObjectMapper mapper = new ObjectMapper();
+        try {
+            return mapper.readValue(json, Trigger.class);
+        } catch (Exception e) {
+            e.printStackTrace();
+            return null;
+        }
+    }
+
+    private byte [] writeJson(Trigger trigger) throws JsonProcessingException {
+        return new ObjectMapper().writeValueAsBytes(trigger);
+    }
+
+    @Test
+    public void testTriggerWithComplexActionValues() throws Exception {
+        String json = "{\"id\":12345678,\"title\":\"Test Trigger\",\"active\":true,\"position\":45,\"conditions\":{\"all\":[],\"any\":[{\"field\":\"comment_includes_word\",\"operator\":\"includes\",\"value\":\"@triggerTest\"}]},\"actions\":[{\"field\":\"notification_target\",\"value\":[\"1234567890\",[[\"key1\",\"value1\"],[\"key2\",\"value2\"]]]}],\"createdAt\":null,\"updatedAt\":null}";
```

Comment:
@Dohbedoh Maybe it's like this in others tests but couldn't we move them to resources files and load them to have a better readability ?

Reply:
Done. The compact `json` is used for the last assertion of each test.